.github/workflows/fly-deploy.yml

```diff
@@ -0,0 +1,18 @@
+# See https://fly.io/docs/app-guides/continuous-deployment-with-github-actions/
+
+name: Fly Deploy
+on:
+  push:
+    branches:
+      - main
```

Comment:
we don't have a main branch currently

Reply:
okay.

I was wondering about this because this file was auto-generated after running `fly launch --no-deploy` and originally had `main`, but I changed it to `master` in the first commit. Then I realized that I did not add `fly.yml`, so I changed it back to `main`, how it originally came.

Anyway, I'll change it back. Haha

Reply:
no worries, I think you can keep the other file and delete this one

Reply:
Seeing this after I pushed the commit.

Reply:
yeah, I don't think we need two files